src/Sarif.Multitool.Library/Rules/SARIF1004.ExpressUriBaseIdsCorrectly.cs

```diff
@@ -55,6 +62,16 @@ protected override void Analyze(ArtifactLocation artifactLocation, string artifa
                     artifactLocation.UriBaseId,
                     artifactLocation.Uri.OriginalString);
             }
+
+            if (!artifactLocationUri.IsAbsoluteUri && artifactLocationUri.OriginalString.StartsWith("/") == true)
```

Comment:
I personally don't like to compare to == true if the method returns a boolean. Not sure if we have this pattern in this repo

Reply:
Good catch. That is left over from when I was using a nullable bool. Fixed.

---
In reply to: [497694414](https://github.com/microsoft/sarif-sdk/pull/2095#discussion_r497694414) [](ancestors = 497694414)